### DISCUSSION TITLE
No Gpu Temp Limit slider for Rtx 30 series in GHELPER?

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1785] [Q&A] [answered]
@seerge why there is no Gpu temp limit slider for rtx 3060...is this option only available for 2022 and 2023 models :(

![20231224_224030](https://github.com/seerge/g-helper/assets/154586512/4adf8950-13c5-4432-88c8-d0d601a86e3e)

**Comment by seerge** [comment, marked as answer]
If it's not there it's not supported by your hardware

**Reply by RAGE-HERO7** [reply]
Oh i see....In that case it is what it is 🙂

But i gotta say after undervolting cpu via GHELPER the gpu temps redused by 83°C
from 86°C 🤝
